usaspending_api/api_contracts/contracts/v2/download/count.md

```diff
@@ -112,8 +112,8 @@ Returns the number of transactions that would be included in a download request
 + `upper_bound`: 1000000 (optional, number)
 
 ### NAICSCodeObject (object)
-+ `require`: [`33`] (optional, array[string], fixed-type)
```

Comment:
What is the reason for removing these brackets?

Reply:
I was trying to fix the Dredd request to send a valid JSON array. This change appeared to fix this filter value, but lead to other issues with the nested arrays below